components/popover_list_members/index.ts

```diff
@@ -2,27 +2,43 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
 
 import {createSelector} from 'reselect';
 
 import {getAllChannelStats} from 'mattermost-redux/selectors/entities/channels';
 import {getCurrentRelativeTeamUrl} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUserId, getUserStatuses, makeGetProfilesInChannel} from 'mattermost-redux/selectors/entities/users';
 import {getTeammateNameDisplaySetting, getAddMembersToChannel} from 'mattermost-redux/selectors/entities/preferences';
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Channel} from 'mattermost-redux/types/channels';
+import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
+
+import {GlobalState} from 'types/store';
 
 import {openDirectChannelToUserId} from 'actions/channel_actions.jsx';
 import {loadProfilesAndStatusesInChannel} from 'actions/user_actions.jsx';
 import {openModal} from 'actions/views/modals';
+
 import {canManageMembers} from 'utils/channel_utils';
 import {sortUsersByStatusAndDisplayName} from 'utils/utils.jsx';
 
-import PopoverListMembers from './popover_list_members.jsx';
+import PopoverListMembers, {Props} from './popover_list_members';
+
+type Filters = {
+    role?: string;
+    inactive?: boolean;
+    active?: boolean;
+    roles?: string[];
+    exclude_roles?: string[];
+    channel_roles?: string[];
+    team_roles?: string[];
+};
 
-const makeSortUsersByStatusAndDisplayName = (doGetProfilesInChannel) => {
+const makeSortUsersByStatusAndDisplayName = (doGetProfilesInChannel: (state: GlobalState, channelId: Channel['id'], filters?: Filters) => UserProfile[]) => {
     return createSelector(
         'makeSortUsersByStatusAndDisplayName',
-        (state, channelId) => doGetProfilesInChannel(state, channelId, true),
+        (state: GlobalState, channelId: Channel['id']) => doGetProfilesInChannel(state, channelId, {active: true}),
```

Comment:
@hmhealey coming back to your previous comment related to changing from `true` to `{active: true}`. You are right, this will actually change the behavior.  

I was able to track down the usage of `filters` in the following places with this order 
- [makeGetProfilesInChannel](https://github.com/mattermost/mattermost-webapp/blob/7759e71afa0ffa94c4e2bec2e1d09537ae0982e5/packages/mattermost-redux/src/selectors/entities/users.ts#L559) and 
- [filterProfiles](https://github.com/mattermost/mattermost-webapp/blob/7759e71afa0ffa94c4e2bec2e1d09537ae0982e5/packages/mattermost-redux/src/selectors/entities/users.ts#L281)

As `filters` is expecting an object, so we can't just pass `true` but we can't also pass `{active: true}` unless we want to change the behavior. I assume when we pass just `true` this has no effect. 

Therefore we can just remove `{active: true}` not pass anything as default value for `filters` is `{}` on [makeGetProfilesInChannel](https://github.com/mattermost/mattermost-webapp/blob/7759e71afa0ffa94c4e2bec2e1d09537ae0982e5/packages/mattermost-redux/src/selectors/entities/users.ts#L559) and it is optional in [filterProfiles](https://github.com/mattermost/mattermost-webapp/blob/7759e71afa0ffa94c4e2bec2e1d09537ae0982e5/packages/mattermost-redux/src/selectors/entities/users.ts#L281)

Please let me know what you think on this?

Reply:
Yeah, I think passing nothing would work best. That'll also work around an issue I noticed in `doGetProfilesInChannel` where it doesn't handle a filters being an object properly, causing it to do a bunch of unnecessary background work. That's a separate bug to fix though

Reply:
Agreed, I did the last commit